src/mmw/sass/pages/_water-balance.scss

```diff
@@ -300,9 +300,13 @@
         float: left;
         position: relative;
         margin: 8px 8px 0 8px;
-        width: 66px;
+        width: 61px;
```

Comment:
So did we accidentally increase the bounding box size or decrease the tile size necessitating this?